src/components/mainMenu/mainMenu.js

```diff
@@ -0,0 +1,47 @@
+import './mainMenu.css';
+
+const PEOPLE_MODE = 'People';
+const VEHICLES_MODE = 'Vehicles';
+const STARSHIPS_MODE = 'Starships';
+
+function mainMenu() {
```

Comment:
Moglabym zasugerowac nazwe typu np. createMainMenu.

Reply:
Zrobione. Masz racje to jest bardziej intuicyjne.